elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/AsyncTest.java

```diff
@@ -39,6 +39,7 @@
                 + "\t\t('com.example.repository','Example Repository','The code for this project', false);")
 @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD,
         statements = "DELETE FROM ArtifactVersion; DELETE FROM ArtifactProduct; DELETE FROM ArtifactGroup;")
+@Import(DBPasswordExtractorSetup.class)
```

Comment:
Why does Async need the DBPassword config?

Reply:
this test uses shared application.yaml where dynamic config is enabled so it tries to parse DB and hjson and create DataSource object.
Without it, it will run fine independently but will fail when all tests are run together for incorrect password as it will use default implementation.

Reply:
this is no longer required after removing db connection hjson